packages/web-react/src/components/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useClassNamePrefix } from '../../hooks';
+import { useStyleProps } from '../../hooks/styleProps';
```

Comment:
couldn't this be imported from hooks too?

Reply:
Yes, thanks, fixed.